### DISCUSSION TITLE
Make NoFail only "activate" when Player actually failed

### DISCUSSION THREAD

**Discussion by NatureHog** [ppy/osu#16779] [Ideas] [answered]
If someone is using NF only as a "backup" i.e.  doesn't know if they will fail or not, it doesn't make sense to half their final score, if they didn't actually fail.

So is it possible, when a player uses NF, to only half the final score if the player actually failed (aka lost all their health)?
Beat Saber actually has something similar, where the NoFail mod in that game **doesn't** have an impact on the final score, _unless_ the player "failed the level", in which case the level keeps going, but the final score is halved.

**Comment by bdach** [comment, marked as answer]
has been rejected several times. doing this would make it so there would be no reason to ever *not* select no fail.

duplicate of: https://github.com/ppy/osu/issues/9844, https://github.com/ppy/osu/discussions/14114

**Reply by NatureHog** [reply]
I see, apologies!
